configure.ac

```diff
@@ -230,6 +230,18 @@ AS_CASE([$with_gc],
     )
 AC_MSG_RESULT([$with_gc])
 
+dnl
+dnl User setting: dynamic library mode (off by default)
+dnl
+AC_ARG_ENABLE([libgap],
+    [AS_HELP_STRING([--enable-libgap], [enable building of dynamic library])],
+    [AC_DEFINE([BUILD_LIBGAP], [1], [define if building libgap])],
+    [enable_libgap=no]
+)
+AC_MSG_CHECKING([whether to enable dynamic library mode])
+AC_MSG_RESULT([$enable_libgap])
+AC_SUBST([BUILD_LIBGAP], [$enable_libgap])
```

Comment:
So... what about your idea to just drop `--enable-libgap`, and setting `IsLIBGAP` in `GAP_Initialize` instead? It seemed sensible to me. @sebasguts any concerns?